app/views/home/index.slim

```diff
@@ -33,3 +32,14 @@ section.card.card-2.mx1.mb1
             div.table-cell.featured-posts.px1
               h5.h5.m0.card-listItem-header=post.title
               p.m0.card-listItem-text=truncate_content(post.content)
+
+section.footer.mx1.mb1
+  ul.list-reset.h6
+    li
```

Comment:
Hmm I feel the `li` should be inside the `for` loop. So we have multiple items (links) in a list.